### DISCUSSION TITLE
I accidentally purchased the wrong domain because of a typo

### DISCUSSION THREAD

**Discussion by anavalo** [vercel/community#4491] [Help] [answered]
### Summary

Please help, I accidentally purchased the wrong domain. What can I do to cancel the transaction?

### Steps to Reproduce

No steps

**Comment by amyegan** [comment, marked as answer]
Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team for help with this (you can use the `Billing` problem area). If you let me know the case number, I can double check the ticket to be sure it gets in front of the right team quickly 🙏
